src/test/java/poll/service/VoteServiceTest.java

```diff
@@ -1,4 +1,142 @@
 package poll.service;
 
-public class VoteServiceTest {
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import poll.repository.Answer;
+import poll.repository.IVoteRepository;
+import poll.repository.Vote;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class VoteServiceTest {
+
+    @Mock
+    private IVoteRepository voteRepository;
+    @InjectMocks
+    private VoteService voteService;
+
+    private List<Vote> answersDb;
+
+    @BeforeEach
+    void init() {
+        answersDb = Arrays.asList(
+                new Vote(1L, "Earvin Ngapeth", LocalDateTime.of(2021, 1, 1, 14, 30), Answer.NO),
+                new Vote(2L, "David Smith", LocalDateTime.of(2021, 1, 1, 15, 30), Answer.YES),
+                new Vote(3L, "Ivan Zaitsev", LocalDateTime.of(2021, 1, 2, 9, 30), Answer.YES),
+                new Vote(4L, "Osmany Juantorena", LocalDateTime.of(2021, 1, 2, 9, 45), Answer.NO),
+                new Vote(5L, "Uros Kovacevic", LocalDateTime.of(2021, 1, 2, 10, 0), Answer.NO),
+                new Vote(6L, "Marko Podrascanin", LocalDateTime.of(2021, 1, 2, 14, 30), Answer.NO),
+                new Vote(7L, "Swan Ngapeth", LocalDateTime.of(2021, 1, 3, 8, 0), Answer.YES),
+                new Vote(8L, "Kamil Semeniuk", LocalDateTime.of(2021, 1, 3, 14, 0), Answer.YES),
+                new Vote(9L, "Seyed Mousavi", LocalDateTime.of(2021, 1, 3, 14, 59), Answer.YES)
+        );
+    }
+
+
+    @Test
+    void should_find_vote_by_id() {
+        //given
+        long id = 1L;
+        when(voteRepository.findById(id)).thenReturn(Optional.of(answersDb.get(0)));
+        //when
+        //then
+        assertThat(answersDb.get(0)).isEqualTo(voteService.getVoteById(id));
+    }
+
+    @Test
+    void should_throw_exception_when_findById_not_found_Vote() {
```

Comment:
minor: you decided to use the snake case naming convention in all other tests and here there's a mix of snake and camel case :D. We should be consistent and stikc with the one convention

Reply:
I just left it like this, becaouse findById is a method and I decided to do not split the name. Should i change my thinking?

Reply:
Well, the thinking is fine but in general the snake case is not so popular in the Java world. I would use the concept of:
should_expectedBehaviourInCamelCase_when_conditionsInCamelCase. But for now its up to you